ts/components/conversation/Quote.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import classnames from 'classnames';
+
+// @ts-ignore
+import Mime from '../../../js/modules/types/mime';
+
+
+interface Props {
+  attachments: Array<QuotedAttachment>;
+  authorColor: string;
+  authorProfileName?: string;
+  authorTitle: string;
+  i18n: (key: string, values?: Array<string>) => string;
```

Comment:
Nice, your type definition is more full-fledged than what I’ve been using! Will totally steal this 😄